docs/technical_reference/flowsheets/gac.rst

```diff
@@ -0,0 +1,91 @@
+Granular Activated Carbon
+=========================
+
+Introduction
+------------
+
+The simple granular activated carbon (GAC) flowsheet can be simulated to predict the performance of a GAC system to treat residual organics. This flowsheet can
+be useful to expedite the set-up, usage, and costing of of a GAC system for its conventional water treatment applications using the constant pattern homogeneous
+surface diffusion model (CPHSDM) model.
+
+Implementation
+--------------
+
+As a single unit operation, the assumptions for the flowsheet are aligned with those detailed in the :doc:`GAC unit model documentation <../unit_models/gac>`.
```

Comment:
@MarcusHolly I'm going to point you to this because I see you were hard linking some internal documentation hyperlinks in the flowsheet documentation. I think this method is better because it should link to the version of the documentation the reader is using instead of having the hard link to `latest`. But honestly someone more familiar than me with .rst compiling should give a recommendation.

Reply:
So I was looking through the docs to see how it is done to get this one in, and the answer is that it depends. I think all the core files like getting started and how-to's use .rst utilities instead of hard linking, but they rely on establishing a name at the top of certain documentation files (which the unit models don't have but the how-to's and others do) and then referencing the name inline, instead of here which references the file path tree to the desired link.

Reply:
From a quick skim, it looks like some flowsheets, unit models, and one property model documentation are hard-coding links

Reply:
Maybe the best solution it to just change the hard-coded link from `latest `to `stable `because a user might have an outdated version of the documentation, and we would still want the links to re-direct them to more up-to-date documentation. I see that some documentation does this already (`getting_started.rst`, `how_to_setup_simple_chemistrt.rst`, etc.)

Reply:
IMO you would always want the version to stay the same before and after clicking the link which you can't get through hard-coding, but yea this would be a tedious review of the current documentation.